schemas/Object.yaml

```diff
@@ -818,6 +874,42 @@ funcs:
         functor: SinglePrecisionFloatColumn
         args: rho
         dataset: Exp_multiprofit
+    exponential_reff_major:
+        functor: SemimajorAxisFromMoments
```

Comment:
This is more like `SemimajorAxisFromEllipse`, not really moments anymore? I think the more readable solution is to create new derived classes for the conversions from `reff_x`, `reff_y`, which can derive from `MomentsBase` but won't need the extra bare `false` at the end of the parameter list to avoid doing the wrong thing. And then the functors that do take in moments could default `is_covariance` to true, so all the bare `true`'s are avoided in the yaml as well.

Reply:
You can consider them both to be from moments - the distinction is whether the moments are in the form of terms of a covariance matrix or a correlation matrix.

I did something like what you suggested by renaming the previous shape_xx, shape_yy, shape_xy args to shape_1_1, shape_2_2 and shape_1_2 (since they are terms of a matrix, just one with constraints like 1_2 == 2_1), and then having `Correlation` subclasses that set `is_covariance` to False. The arg names are now consistent with the CD args, although unfortunately the order is inconsistent since those go 1_1, 1_2, 2_1, 2_2, but I didn't want to change the existing arg order at this stage, so hopefully we can live with that.